department-of-veterans-affairs/vets-website
# Move place of last separation to service period

## Summary
Fix a form field bug on the Pension Benefits form (21P-527EZ)

### Issue
[Issue in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79052)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the 'Service period' page in the 'Military history' chapter
3. Verify that the 'Place of your last separation' field is on the page and is not required
4. Continue to the 'Other service names' page
5. Verify that the 'Place of your last separation' field is not on the page
6. Verify that the data is saved to the api and redux
7. Continue to the 'Review application' page
8. Verify that the data is viewable and editable
9. Verify that the application is able to be submitted

## Testing done
- [x] Updated existing unit tests

## Screenshots
### Single Page Array
| Before | After | After |
| ------ | ------ | ------ |
|![Screenshot 2024-03-22 at 12 01 02 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/4cba2ac8-5a6f-4890-b459-ac99707b64c1)|![localhost_3001_pension_application_527EZ_form-saved (25)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/01de52a5-0f61-4e36-a9f6-e1ed754076d5)|![localhost_3001_pension_application_527EZ_form-saved (24)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/1f9a25b9-79ef-4886-a0df-851809834a4a)

## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling

### Authentication
